Dockerfile

```diff
@@ -1,4 +1,7 @@
-FROM mcr.microsoft.com/dotnet/sdk:7.0.404-1-bullseye-slim-amd64
+FROM mcr.microsoft.com/dotnet/sdk:8.0
```

Comment:
Should we not use the bullseye slim version anymore?

Reply:
bullseye slim is debian 11 based, this one is based on the newest debian 12

Reply:
I thought slim was part of the bullseye name